e2e/utils/graphql.ts

```diff
@@ -33,10 +34,16 @@ const createGaloyServerClient = (config) => (authToken) => {
   })
 }
 
-const authToken = process.env.GALOY_TOKEN
-const authTokenOther = process.env.GALOY_TOKEN_2
+const randomizeTokens = (arr: string[]): string => {
+  const randomIndex = Math.floor(Math.random() * arr.length)
+  return arr[randomIndex]
+}
+
+const authTokens = process.env.GALOY_TEST_TOKENS?.split(",") || []
+const secondAuthToken = process.env.GALOY_TOKEN_2 || ""
+export const firstAuthToken = randomizeTokens(authTokens)
```

Comment:
I would rename these to be more specific,  such as `mobileUserToken` and secondAuthToken to be `recieverToken`